requirements_all.txt

```diff
@@ -1898,7 +1898,7 @@ python-gitlab==1.6.0
 python-hpilo==4.3
 
 # homeassistant.components.izone
-python-izone==1.1.8
+python-izone==1.2.3
```

Comment:
I don't find a github release or tag for version 1.2.3. Is it possible to add that so we can see the changes?

Reply:
Oh yeh sure. I think I forgot to push the tag. Will do this when I get home. It's the head version anyhow.

Reply:
Thanks!